opensrp-chw/src/main/java/org/smartregister/chw/activity/HivRegisterActivity.java

```diff
@@ -0,0 +1,80 @@
+package org.smartregister.chw.activity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import com.google.android.material.bottomnavigation.LabelVisibilityMode;
+
+import org.jetbrains.annotations.NotNull;
+import org.smartregister.chw.core.activity.CoreHivRegisterActivity;
+import org.smartregister.chw.core.custom_views.NavigationMenu;
+import org.smartregister.chw.fragment.HivFollowupRegisterFragment;
+import org.smartregister.chw.fragment.HivRegisterFragment;
+import org.smartregister.chw.hiv.fragment.BaseHivCommunityFollowupRegisterFragment;
+import org.smartregister.chw.hiv.fragment.BaseHivRegisterFragment;
+import org.smartregister.helper.BottomNavigationHelper;
+import org.smartregister.listener.BottomNavigationListener;
+
+import java.util.List;
+
+public class HivRegisterActivity extends CoreHivRegisterActivity {
+    private String baseEntityID;
```

Comment:
Codacy found an issue: [Perhaps 'baseEntityID' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5590665)